### ISSUE TITLE
tag all uploads

### ISSUE THREAD

**Issue by umlaeute** [electrickery/pd-sigpack#3]
please tag all uploads to deken.

E.g. there's a [sigpack/0.0.45](http://deken.puredata.info/library/sigpack/0.0.45) upload, but no corresponding tag in this repository.

also i'm not exactly sure about the versioning scheme.
is it `0.045` or `0.0.45`?
this is somewhat important as it influences how packages are sorted. e.g. deken now features both `0.0.45` and `0.044`, where the latter is considered newer than the former, as `0.0.45 << 0.44(.0)`.
the upload of `0.044` basically burned the `0.0.x` versioning scheme irreversibly.

**Comment by electrickery** [comment]
Hi,
A tag is added, hopefully in the correct way.
IIRC*, 0.045 was the original format used in sigpack, which I moved to 0.0.45 to make it compliant with the Major.Minor.Patch scheme. Added another tag "v0.0.45", assuming this is the correct format (same as in deken?).
*) WARNING: it was a long time ago...

**Comment by umlaeute** [comment]
indeed, it was a long time ago (i just decided to update the Debian packages to the latest and greatest versions, that's why i came across this).

the problem remains that deken interprets a version string `v0.044`  as `0.44` (`major.minor`, with ~~`patch`~~ being implicitly set to `0`), which sorts after `0.0.45` (`major.minor.patch`), since the `major` version is the same (**0**) and the `minor` version **45** is clearly after **0**.

to guarantee stable sorting, i think it would be best to consistently bump the `minor` version to (e.g.) **45** and drop the `0.0.X` versions (you can of course still use patch-versions, as in `0.45.1`)

as for `vX.Y.Z`  vs `X.Y.Z` i don't really care very much. personally I prefer using the `v`-prefix, but that's up to taste. it owuld be great if it was consistent though :-)

**Comment by electrickery** [comment]
Fixed this for all my pd-related repositories.
